frontend/src/scenes/sessions/SessionsView.tsx

```diff
@@ -213,6 +210,7 @@ export function SessionsView({ personIds, isPersonPage = false }: SessionsTableP
             </div>
 
             <Table
+                className="ph-no-capture"
```

Comment:
Don't think sanitizing the whole table is necessary, how about the Person & Event columns?